### ISSUE TITLE
Mismatch between Cocoapods and Repo code?

### ISSUE THREAD

**Issue by aravasio** [paulz/ImageCoordinateSpace#4]
I'm seeing _huge_ differences between the code available in Cocoapods and the repo here in GH. Maybe Swift3 code??

Could there be an update pending? I'm interested in using this lib to skip doing some manual transformations, but as it is it won't build out of the box.

An example is
```
class TransformedCoordinateSpace : NSObject, UICoordinateSpace {
...
}
```

This class declaration is already different than what the repo in GH has, where declaration is `ClassName: NSObject` and the second extension is handled in an `extension` tag right below.

https://github.com/paulz/ImageCoordinateSpace/blob/master/ImageCoordinateSpace/TransformedCoordinateSpace.swift

**Comment by paulz** [comment]
Cocoapods releases are made by tags, see latest example: 
https://github.com/CocoaPods/Specs/blob/master/Specs/4/2/4/ImageCoordinateSpace/1.0.5/ImageCoordinateSpace.podspec.json#L15

While the link to GitHub in this issue points to master. Here is the correct link using the tag: https://github.com/paulz/ImageCoordinateSpace/blob/1.0.5/ImageCoordinateSpace/TransformedCoordinateSpace.swift

**Comment by paulz** [comment]
Also, I've just released latest version 1.0.6 that supports Swift 4.1 and Xcode 9.3, see https://github.com/CocoaPods/Specs/blob/master/Specs/4/2/4/ImageCoordinateSpace/1.0.6/ImageCoordinateSpace.podspec.json#L16

Closing
